services/workflows-service/src/case-management/case-management.service.ts

```diff
@@ -69,4 +78,157 @@ export class CaseManagementService {
       throw ValidationError.fromAjvError(validate.errors!);
     }
   }
+
+  async createUbo({
+    workflowId,
+    ubo,
+    projectId,
+  }: {
+    workflowId: string;
+    ubo: Record<string, any>;
+    projectId: TProjectId;
+  }) {
+    await this.prismaService.$transaction(async transaction => {
+      const workflowRuntimeData =
+        await this.workflowService.getWorkflowRuntimeDataByIdAndLockUnscoped({
+          id: workflowId,
+          transaction,
+        });
+
+      const uboToEntityAdapter = (ubo => {
+        return {
+          id: randomUUID(),
+          type: 'individual',
+          variant: 'ubo',
+          data: {
+            firstName: ubo.firstName,
+            lastName: ubo.lastName,
+            email: ubo.email,
+            nationalId: ubo.nationalId,
+            percentageOfOwnership: ubo.ownershipPercentage ?? ubo.percentageOfOwnership,
+            additionalInfo: {
+              fullAddress: ubo.fullAddress,
+              nationality: ubo.nationality,
+              companyName: workflowRuntimeData.context.entity.data.companyName,
+              customerCompany:
+                workflowRuntimeData.context.collectionFlow.additionalInformation.customerCompany,
+            },
+          },
+        };
+      }) satisfies UboToEntityAdapter;
+
+      const [{ ballerineEntityId }] = await this.workflowService.createOrUpdateWorkflowRuntime(
+        {
+          workflowDefinitionId: kycEmailSessionDefinition().id,
```

Comment:
Hard-coding the KYC workflow is problematic - as some customers use a different definition.
I think the definition ID should be set in the workflow definition's config, and if it is not set then we should perform an early exit and raise an error.